source/tutorials/sample-classifier.rst

```diff
@@ -39,21 +54,74 @@ Next, we will attempt to predict which body site each sample originated from bas
      --p-optimize-feature-selection \
      --p-parameter-tuning \
      --p-estimator RandomForestClassifier \
-     --p-n-estimators 100 \
+     --p-n-estimators 20 \
      --output-dir moving-pictures-classifier
 
-The visualization produced by this command presents classification accuracy results in the form of a confusion matrix. This matrix indicates how frequently a sample is classified with to the correct class vs. all other classes. The confusion matrix is displayed at the top of the visualization in the form of a heatmap, and below that as a table containing overall accuracy (the fraction of times that test samples are assigned the correct class).
+
+This pipeline produces several outputs. First let's check out ``accuracy_results.qzv``, which presents classification accuracy results in the form of a confusion matrix. This matrix indicates how frequently a sample is classified with to the correct class vs. all other classes. The confusion matrix is displayed at the top of the visualization in the form of a heatmap, and below that as a table containing overall accuracy (the fraction of times that test samples are assigned the correct class).
```

Comment:
_a sample is classified with to the correct class _ -> _a sample is classified with the correct class _